froide/comments/templatetags/froide_comment_tags.py

```diff
@@ -13,3 +13,10 @@ def get_froide_comment_form(context, object):
         initial["name"] = request.user.get_full_name()
 
     return CommentForm(object, initial=initial)
+
+
+@register.simple_tag(takes_context=True)
+def is_moderator(context):
+    request = context["request"]
+    user = request.user
+    return user.groups.filter(permissions__codename="moderate").exists()
```

Comment:
Likely a query can be avoided (because the result will already be cached on the user) by using the [standard permission API of Django `user.has_perm`](https://docs.djangoproject.com/en/4.0/topics/auth/default/#permission-caching) instead of a custom ORM query.

Reply:
Also instead of a custom template tag, either use the existing `can_moderate_foirequest` (that has caching) or try `{{ perms.foirequest.moderate }}` (see [docs](https://docs.djangoproject.com/en/4.0/topics/auth/default/#permissions)).

Reply:
As far as I understood `can_moderate_foirequest ` and the `is_moderator` annotation on the comment don't behave the same. Which one should we use? If we use `can_moderate_foirequest ` staff users would also get the badge even if their are not in an moderation group (I think)